### PR TITLE
clear Send password if whitespace

### PR DESCRIPTION
**Pull request by addisonbeck**
Resolves https://app.asana.com/0/1199659118818122/1199987076477970/f

Sibling to https://github.com/bitwarden/jslib/pull/287

### CODE REVIEW

File: src/App/Pages/Send/SendAddEditPageViewModel.cs [hunk 1]
```diff
@@ -337,6 +337,11 @@ public async Task<bool> SubmitAsync()
 
             UpdateSendData();
 
+            if (NewPassword != null && string.IsNullOrWhiteSpace(NewPassword))
```
Comment (cscharf) [inline src/App/Pages/Send/SendAddEditPageViewModel.cs:340]: You can remove the bit of `NewPassword != null && `, `string.IsNullOrWhiteSpace()` already handles the `null` condition for you.

Comment (addisonbeck) [reply src/App/Pages/Send/SendAddEditPageViewModel.cs:340]: In this case though, null is the value we are after. I'm just using IsNullOrWhiteSpace() for the empty/whitespace catch. If NewPassword is already null we don't need to null it out.

Comment (cscharf) [reply src/App/Pages/Send/SendAddEditPageViewModel.cs:340]: setting `NewPassword` to `null` if it's already `null`... no harm, no foul. The `if` check just seems a little too complicated/confusing perhaps... maybe it's just me 🤷🏻

Comment (addisonbeck) [reply src/App/Pages/Send/SendAddEditPageViewModel.cs]: I def see either way. Went ahead and removed it. Less code is usually better code.

**Review by cscharf — APPROVED** [review]

**Pull request merged**